packages/amplify_datastore/example/lib/Post.dart

```diff
@@ -0,0 +1,93 @@
+import 'package:amplify_datastore_plugin_interface/amplify_datastore_plugin_interface.dart';

```

Comment:
Not sure if we need license boilerplate for example apps?

Reply:
we should in my opinion. Adding here.